test/replicator.spec.js

```diff
@@ -45,7 +45,7 @@ Object.keys(testAPIs).forEach((IPFS) => {
       log = new Log(ipfs, testIdentity)
 
       store = new DummyStore(log, ipfs, testIdentity)
-      replicator = new Replicator(store, 123)
+      replicator = new Replicator(store, 10)
```

Comment:
Here I changed the replicator's concurrency to match the log's pointerCoint. Wondering also here if we could just use the log's pointerCount as concurrency, or potentially just max this out?